docs/calculations.mdx

```diff
@@ -427,7 +428,7 @@ creative_consumer_device_usage_emissions_gco2pm =
       lookup_carbon_intensity_gco2e_per_kwh(country, region, utc_datetime)
 
 creative_consumer_device_embodied_emissions_gco2pm =
```

Comment:
@bokelley Is this per impression, so in final calculations should we be multiplying by the number of impressions in the request input? Or does number of impressions only affect calculating video bytes? 

Likewise for `creative_consumer_device_usage_emissions_gco2pm`, `creative_data_transfer_usage_emissions_gco2pm`, `creative_data_transfer_embodied_emissions_gco2pm`